src/classes/User/User.ts

```diff
@@ -109,6 +125,10 @@ export class User extends PassageBase {
      * @return {Promise<PassageUser>} Passage User object
      */
     public async update(userId: string, options: UpdateUserArgs): Promise<PassageUser> {
+        if (!userId) {
+            throw new Error('userId is required.');
+        }
```

Comment:
Need a check for options.

Reply:
in this case options must be passed in or else it won't transpile. this differs from ruby where it is not type hinted and the options arg has a default value